pkgs/os-specific/linux/pcmciautils/default.nix

```diff
@@ -1,10 +1,9 @@
-{ stdenv, fetchurl
+{ config, lib, stdenv, fetchurl
 , yacc, flex
 , sysfsutils, kmod, udev
-, firmware # Special pcmcia cards.
-, config   # Special hardware (map memory & port & irq)
-, lib      # used to generate postInstall script.
-}:
+, firmware   ? config.pcmciaUtils.firmware or [] # Special pcmcia cards.
+, configOpts ? config.pcmciaUtils.config or null # Special hardware (map memory & port & irq)
+}:                   # used to generate postInstall script.
```

Comment:
I wonder how to find out how many users will be hit by such an incompatible rename.